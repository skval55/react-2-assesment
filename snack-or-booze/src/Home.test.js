import { render, screen } from "@testing-library/react";
import Home from "./Home";
import React from "react";

// smoke test see if it renders
test("see if it renders", () => {
  render(<Home snacks={"snacks"} drinks={"drinks"} />);
});
test("see if it renders", () => {
  const { asFragment } = render(<Home snacks={"snacks"} drinks={"drinks"} />);
  expect(asFragment()).toMatchSnapshot();
});
