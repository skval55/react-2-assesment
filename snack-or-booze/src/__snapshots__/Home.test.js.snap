// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`see if it renders 1`] = `
<DocumentFragment>
  <section
    class="col-md-8"
  >
    <div
      class="card"
    >
      <div
        class="text-center card-body"
      >
        <h5
          class="card-title"
        />
        <h3
          class="font-weight-bold"
        >
          Welcome to Silicon Valley's premier dive cafe!
        </h3>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
